BridgeAppSDKSample/Base.lproj/consent_19sharing_rsch.html

```diff
@@ -17,7 +17,7 @@
 
 <ul style="padding-left:20px;">
 <li>The US Department of Health and Human Services (HHS), the Office for Human Research Protection (OHRP), the Food and Drug Administration (FDA), and other agencies for review of our research procedures.</li>
-<li><import href="consent_irb.html" /> so they can monitor the safety, effectiveness, and conduct of our research.</li>
+<li>The Institutional Review Board (IRB) so they can monitor the safety, effectiveness, and conduct of our research.</li>
```

Comment:
Um? Why would you need to change the sample project consent documents? Also, this is here b/c when we were doing the Luxembourg project, they have a different IRB.